src/API/API.ts

```diff
@@ -1,7 +1,15 @@
-export const getRandomName = (length: number) => {
-  return fetch(`http://uinames.com/api/?minlen=${length}&maxlen=${length}`)
-    .then(res => res.json())
-    .then(res => {
-      return `${res.name} ${res.surname}`;
-    }) as Promise<string>;
-};
+export const getRestaurants = (location: string, radius: number) => {
```

Comment:
this file should use axios to build query instead of fetch, example:

```
import axios, { AxiosError } from 'axios';
export default function getRoutes(config: m.RouteConfig) {
	return {
		getById: function({ id }: { id: number }): Promise<m.PafQuery | null> {
      return axios({
        method: 'get',
        url: `${config.pafQueryEndpoint}/pafQuery/getById`,
        params: { id },
        data: {},
        headers: {
          'Content-Type': 'application/json',
          Pragma: 'no-cache',
          'Cache-Control': 'no-cache, no-store'
        },
        timeout: config.timeout
      }).then(
        res => valueOrThrow(t.union([m.PafQuery, t.null]), config.unwrapApiResponse(res.data)),
        parseError
      ) as any;
    },
    getQueryCats: function(): Promise<Array<string>> {
      return axios({
        method: 'get',
        url: `${config.pafQueryEndpoint}/pafQuery/getQueryCats`,
        params: {},
        data: {},
        headers: {
          'Content-Type': 'application/json',
          Pragma: 'no-cache',
          'Cache-Control': 'no-cache, no-store'
        },
        timeout: config.timeout
      }).then(
        res => valueOrThrow(t.array(t.string), config.unwrapApiResponse(res.data)),
        parseError
      ) as any;
    },
	}
}

```

Reply:
then, in file API.ts:

```
import * as config from 'config';
import getPafQueryRoutes from 'contracts/queryApi';
const pafQuery = getPafQueryRoutes(routeConfigContracts);

export default { pafQuery };
```

wasn't structured like this in the demo app? we could refactor this in pairing